app/controllers/integrations/base_controller.rb

```diff
@@ -45,10 +45,14 @@ def deploy?
     raise NotImplementedError, "#deploy? must be overridden in a subclass"
   end
 
+  def release_params
```

Comment:
then maybe some comment here `# overwritten by external plugin` ... maybe just contribute buildkite support to samson core ... sounds kind of useful ... or is it somewhat specific to your use ?